geode-core/src/distributedTest/java/org/apache/geode/cache/CreateAndLocalDestroyInTXRegressionTest.java

```diff
@@ -67,7 +67,7 @@
   private VM otherVM;
 
   @ClassRule
-  public static DistributedTestRule distributedTestRule = new DistributedTestRule();
+  public static DistributedRule distributedTestRule = new DistributedRule();
```

Comment:
do you want to rename these static fields names to "distributedRule"? It is in a large number of files

Reply:
I wasn't planning to but every class containing the old variable name was touched anyway, so I probably should go ahead and do it. Thanks!